[2024-06-27T19:12:23.990+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T19:12:24.032+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:11:32.912358+00:00 [queued]>
[2024-06-27T19:12:24.045+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:11:32.912358+00:00 [queued]>
[2024-06-27T19:12:24.045+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T19:12:24.064+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): run_silver_job> on 2024-06-27 19:11:32.912358+00:00
[2024-06-27T19:12:24.072+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=888) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-27T19:12:24.073+0000] {standard_task_runner.py:63} INFO - Started process 889 to run task
[2024-06-27T19:12:24.072+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Stage_1', 'run_silver_job', 'manual__2024-06-27T19:11:32.912358+00:00', '--job-id', '431', '--raw', '--subdir', 'DAGS_FOLDER/kafka_dag.py', '--cfg-path', '/tmp/tmp7yad_w27']
[2024-06-27T19:12:24.074+0000] {standard_task_runner.py:91} INFO - Job 431: Subtask run_silver_job
[2024-06-27T19:12:24.090+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-27T19:12:24.125+0000] {task_command.py:426} INFO - Running <TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:11:32.912358+00:00 [running]> on host bafdbafde709
[2024-06-27T19:12:24.238+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Stage_1' AIRFLOW_CTX_TASK_ID='run_silver_job' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T19:11:32.912358+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-27T19:11:32.912358+00:00'
[2024-06-27T19:12:24.239+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T19:12:24.276+0000] {docker.py:366} INFO - Starting docker container from image bitnami/spark:latest
[2024-06-27T19:12:24.280+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-27T19:12:24.734+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.73 [0m[38;5;2mINFO [0m ==>
[2024-06-27T19:12:24.735+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.73 [0m[38;5;2mINFO [0m ==> [1mWelcome to the Bitnami spark container[0m
[2024-06-27T19:12:24.737+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.73 [0m[38;5;2mINFO [0m ==> Subscribe to project updates by watching [1mhttps://github.com/bitnami/containers[0m
[2024-06-27T19:12:24.739+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.73 [0m[38;5;2mINFO [0m ==> Submit issues and feature requests at [1mhttps://github.com/bitnami/containers/issues[0m
[2024-06-27T19:12:24.741+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.74 [0m[38;5;2mINFO [0m ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit [1mhttps://bitnami.com/enterprise[0m
[2024-06-27T19:12:24.743+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:12:24.74 [0m[38;5;2mINFO [0m ==>
[2024-06-27T19:12:24.753+0000] {docker.py:436} INFO - 
[2024-06-27T19:12:28.310+0000] {docker.py:436} INFO - :: loading settings :: url = jar:file:/opt/bitnami/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-06-27T19:12:28.404+0000] {docker.py:436} INFO - Ivy Default Cache set to: /opt/bitnami/spark/.ivy2/cache
The jars for the packages stored in: /opt/bitnami/spark/.ivy2/jars
[2024-06-27T19:12:28.413+0000] {docker.py:436} INFO - org.postgresql#postgresql added as a dependency
[2024-06-27T19:12:28.414+0000] {docker.py:436} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-5472a397-d843-4ff4-8518-aa6d52176b22;1.0
[2024-06-27T19:12:28.415+0000] {docker.py:436} INFO - confs: [default]
[2024-06-27T19:12:30.927+0000] {docker.py:436} INFO - found org.postgresql#postgresql;42.2.2 in central
[2024-06-27T19:12:30.961+0000] {docker.py:436} INFO - downloading https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.2/postgresql-42.2.2.jar ...
[2024-06-27T19:12:31.153+0000] {docker.py:436} INFO - [SUCCESSFUL ] org.postgresql#postgresql;42.2.2!postgresql.jar(bundle) (212ms)
[2024-06-27T19:12:31.155+0000] {docker.py:436} INFO - :: resolution report :: resolve 2524ms :: artifacts dl 218ms
[2024-06-27T19:12:31.158+0000] {docker.py:436} INFO - :: modules in use:
	org.postgresql#postgresql;42.2.2 from central in [default]
	---------------------------------------------------------------------
	|                  |            modules            ||   artifacts   |
[2024-06-27T19:12:31.159+0000] {docker.py:436} INFO - |       conf       | number| search|dwnlded|evicted|| number|dwnlded|
	---------------------------------------------------------------------
[2024-06-27T19:12:31.162+0000] {docker.py:436} INFO - |      default     |   1   |   1   |   1   |   0   ||   1   |   1   |
	---------------------------------------------------------------------
[2024-06-27T19:12:31.167+0000] {docker.py:436} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-5472a397-d843-4ff4-8518-aa6d52176b22
[2024-06-27T19:12:31.169+0000] {docker.py:436} INFO - confs: [default]
[2024-06-27T19:12:31.180+0000] {docker.py:436} INFO - 1 artifacts copied, 0 already retrieved (771kB/12ms)
[2024-06-27T19:12:31.566+0000] {docker.py:436} INFO - 24/06/27 19:12:31 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-06-27T19:12:32.860+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO SparkContext: Running Spark version 3.5.1
[2024-06-27T19:12:32.861+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO SparkContext: OS info Linux, 6.5.0-41-generic, amd64
[2024-06-27T19:12:32.862+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO SparkContext: Java version 17.0.11
[2024-06-27T19:12:32.954+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO ResourceUtils: ==============================================================
[2024-06-27T19:12:32.954+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-06-27T19:12:32.964+0000] {docker.py:436} INFO - 24/06/27 19:12:32 INFO ResourceUtils: ==============================================================
24/06/27 19:12:32 INFO SparkContext: Submitted application: PostgreSQL Connection with PySpark
[2024-06-27T19:12:33.048+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-06-27T19:12:33.079+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO ResourceProfile: Limiting resource is cpu
[2024-06-27T19:12:33.082+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-06-27T19:12:33.230+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SecurityManager: Changing view acls to: spark
[2024-06-27T19:12:33.231+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SecurityManager: Changing modify acls to: spark
[2024-06-27T19:12:33.232+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SecurityManager: Changing view acls groups to:
[2024-06-27T19:12:33.233+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SecurityManager: Changing modify acls groups to:
[2024-06-27T19:12:33.234+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: spark; groups with view permissions: EMPTY; users with modify permissions: spark; groups with modify permissions: EMPTY
[2024-06-27T19:12:33.633+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO Utils: Successfully started service 'sparkDriver' on port 44021.
[2024-06-27T19:12:33.675+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SparkEnv: Registering MapOutputTracker
[2024-06-27T19:12:33.732+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SparkEnv: Registering BlockManagerMaster
[2024-06-27T19:12:33.773+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
24/06/27 19:12:33 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-06-27T19:12:33.781+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-06-27T19:12:33.824+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-be70ac7f-398c-47be-ae6a-68ed44c981a4
[2024-06-27T19:12:33.852+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-06-27T19:12:33.872+0000] {docker.py:436} INFO - 24/06/27 19:12:33 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-06-27T19:12:34.053+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-06-27T19:12:34.161+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-06-27T19:12:34.235+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar at spark://localhost:44021/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719515552844
[2024-06-27T19:12:34.239+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar at file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719515552844
[2024-06-27T19:12:34.243+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar to /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:12:34.346+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Starting executor ID driver on host localhost
[2024-06-27T19:12:34.347+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: OS info Linux, 6.5.0-41-generic, amd64
[2024-06-27T19:12:34.349+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Java version 17.0.11
[2024-06-27T19:12:34.359+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-06-27T19:12:34.360+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@1fde5ff8 for default.
[2024-06-27T19:12:34.378+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Fetching file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719515552844
[2024-06-27T19:12:34.403+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: /opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar has been previously copied to /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:12:34.411+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Fetching spark://localhost:44021/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719515552844
[2024-06-27T19:12:34.488+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO TransportClientFactory: Successfully created connection to localhost/127.0.0.1:44021 after 54 ms (0 ms spent in bootstraps)
[2024-06-27T19:12:34.496+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: Fetching spark://localhost:44021/jars/org.postgresql_postgresql-42.2.2.jar to /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/fetchFileTemp6863405619950157034.tmp
[2024-06-27T19:12:34.538+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/fetchFileTemp6863405619950157034.tmp has been previously copied to /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:12:34.544+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Executor: Adding file:/tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/userFiles-6467fd39-b6b8-4a55-9b91-36f188ca29fb/org.postgresql_postgresql-42.2.2.jar to class loader default
[2024-06-27T19:12:34.555+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 39301.
[2024-06-27T19:12:34.556+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO NettyBlockTransferService: Server created on localhost:39301
[2024-06-27T19:12:34.573+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-06-27T19:12:34.582+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, localhost, 39301, None)
[2024-06-27T19:12:34.587+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO BlockManagerMasterEndpoint: Registering block manager localhost:39301 with 434.4 MiB RAM, BlockManagerId(driver, localhost, 39301, None)
[2024-06-27T19:12:34.590+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, localhost, 39301, None)
[2024-06-27T19:12:34.592+0000] {docker.py:436} INFO - 24/06/27 19:12:34 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, localhost, 39301, None)
[2024-06-27T19:12:35.160+0000] {docker.py:436} INFO - 2024-06-27 19:12:35,159:create_spark_session:INFO:Spark session created successfully
[2024-06-27T19:12:35.171+0000] {docker.py:436} INFO - 24/06/27 19:12:35 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-06-27T19:12:35.175+0000] {docker.py:436} INFO - 24/06/27 19:12:35 INFO SharedState: Warehouse path is 'file:/opt/bitnami/spark/spark-warehouse'.
[2024-06-27T19:12:38.316+0000] {docker.py:436} INFO - 2024-06-27 19:12:38,315:silver_load:INFO:Load silver table ...
[2024-06-27T19:12:39.431+0000] {docker.py:436} INFO - Traceback (most recent call last):
  File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 75, in <module>
[2024-06-27T19:12:39.432+0000] {docker.py:436} INFO - write_to_postgres()
  File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 71, in write_to_postgres
[2024-06-27T19:12:39.433+0000] {docker.py:436} INFO - silver_load(df,spark)
[2024-06-27T19:12:39.434+0000] {docker.py:436} INFO - File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 63, in silver_load
[2024-06-27T19:12:39.434+0000] {docker.py:436} INFO - df.write.jdbc(POSTGRES_URL, "silver_grocery_purchase", "append", properties=POSTGRES_PROPERTIES)
  File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 1984, in jdbc
  File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
  File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 185, in deco
[2024-06-27T19:12:39.435+0000] {docker.py:436} INFO - pyspark.errors.exceptions.captured.AnalysisException
[2024-06-27T19:12:39.445+0000] {docker.py:436} INFO - : Column item_category not found in schema Some(StructType(StructField(id,IntegerType,false),StructField(item_name,StringType,true),StructField(date_of_purchase,DateType,true),StructField(cost,DecimalType(10,2),true),StructField(quantity,IntegerType,true),StructField(purchased_by,StringType,true),StructField(payment_method,StringType,true),StructField(purchased_category,StringType,true))).
[2024-06-27T19:12:39.446+0000] {docker.py:436} INFO - 2024-06-27 19:12:39,444:close:INFO:Closing down clientserver connection
[2024-06-27T19:12:39.561+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO SparkContext: Invoking stop() from shutdown hook
[2024-06-27T19:12:39.562+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-06-27T19:12:39.582+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO SparkUI: Stopped Spark web UI at http://localhost:4040
[2024-06-27T19:12:39.599+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-06-27T19:12:39.611+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO MemoryStore: MemoryStore cleared
[2024-06-27T19:12:39.613+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO BlockManager: BlockManager stopped
[2024-06-27T19:12:39.623+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-06-27T19:12:39.627+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-06-27T19:12:39.637+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO SparkContext: Successfully stopped SparkContext
[2024-06-27T19:12:39.638+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO ShutdownHookManager: Shutdown hook called
[2024-06-27T19:12:39.638+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO ShutdownHookManager: Deleting directory /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b/pyspark-b7060b4d-911a-42ea-9018-dc75d06e5865
[2024-06-27T19:12:39.645+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO ShutdownHookManager: Deleting directory /tmp/spark-269f83d7-46c9-43df-9b25-a280aa1aa43c
[2024-06-27T19:12:39.651+0000] {docker.py:436} INFO - 24/06/27 19:12:39 INFO ShutdownHookManager: Deleting directory /tmp/spark-bc184756-55fb-4af7-b7e0-31ce3b25c41b
[2024-06-27T19:12:39.930+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T19:12:39.947+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 265, in _raise_for_status
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://docker-proxy:2375/v1.45/containers/create

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 371, in _run_image
    return self._run_image_with_mounts([*self.mounts, tmp_mount], add_tmp_variable=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 398, in _run_image_with_mounts
    self.container = self.cli.create_container(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py", line 439, in create_container
    return self.create_container_from_config(config, name, platform)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py", line 456, in create_container_from_config
    return self._result(res, True)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 271, in _result
    self._raise_for_status(response)
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 267, in _raise_for_status
    raise create_api_error_from_http_exception(e) from e
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/errors.py", line 39, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation) from e
docker.errors.APIError: 400 Client Error for http://docker-proxy:2375/v1.45/containers/create: Bad Request ("invalid mount config for type "bind": bind source path does not exist: /tmp/airflowtmpfwe0ovu4")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 509, in execute
    return self._run_image()
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 380, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 444, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2024-06-27T19:12:39.965+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Stage_1, task_id=run_silver_job, run_id=manual__2024-06-27T19:11:32.912358+00:00, execution_date=20240627T191132, start_date=20240627T191224, end_date=20240627T191239
[2024-06-27T19:12:39.990+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 431 for task run_silver_job (Docker container failed: {'StatusCode': 1}; 889)
[2024-06-27T19:12:40.017+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-27T19:12:40.048+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T19:12:40.051+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
